app/grants/templates/grants/new-whitelabel.html

```diff
@@ -0,0 +1,208 @@
+                  </div>
+                </div>
+
+                <div class="form__group-horizontal pb-3">
+                  <label class="font-body" for="input_title">{% trans "Project Title" %}</label>
+                  <input class="form__input form__input-lg " maxlength="150" type="text" id="input_title" name="input_title" placeholder="The Human Fund" required />
+                </div>
+
+                <div class="form__group-horizontal pb-3">
+                  <label class="font-body" for="input-description">{% trans "Project Description" %}</label>
+                  <div id="input-description">
+                  </div>
+                </div>
+
+                <div class="form__group-horizontal pb-3">
+                  <label class="font-body" for="input-url">{% trans "Project Website" %}</label>
+                  <input type="url" class="form__input form__input-lg " id="input-url" name="reference_url" placeholder="https://human.fund/" required/>
+                </div>
+
+                <div class="form__group-horizontal pb-3">
+                  <label class="font-body" for="input-team_members">{% trans "Project Team members" %}</label>
+                  <div class="form__select2 g-multiselect">
+                    <select id="input-team_members" class="form__input form__input-lg team_members" name="team_members[]" multiple="multiple">
+                    </select>
+                  </div>
+                </div>
+
+                <div class="form__group-horizontal pb-3">
+                  <label class="font-body" for="input-handle2">{% trans "Project Twitter Handle" %}</label>
+                  <input type="text" class="form__input form__input-lg " id="input-handle1" name="handle1" placeholder="@humanfund" />
+                </div>
+
+                <div class="form__group-horizontal pb-3">
+                  <label class="font-body" for="input-handle2">{% trans "Your Twitter Handle" %}</label>
+                  <input type="text" class="form__input form__input-lg " id="input-handle2" name="handle2" placeholder="@georgecostanza"/>
+                </div>
+<!--
+                <h5 class="mt-4">Categorization Information</h5>
+                <hr>
+               <div class="form__group-horizontal pb-3">
+                  <label class="font-body" for="input-grant_type">{% trans "Grant Category" %}</label>
+                  <div class="form__select2">
+                    <select class="js-select2 form__input form__input-lg" name="input-grant_type" id="input-grant_type">
+                      <option selected value="tech">{% trans "Tech" %}</option>
+                      <option value="media">{% trans "Media" %}</option>
+                      <option value="health">{% trans "Public Health" %}</option>
+                    </select>
+                  </div>
+                </div>
+-->
+                <input type="hidden" id="input-grant_type" name="input-grant_type" value="matic">
```

Comment:
is it worth DRY-ing this up, so that there arent copy/paste code anymore?

Reply:
@owocki maybe... id like to see what other revisions we do and how extensible this ends up being - I have a feeling this will sneak back in soon